deployment/packer/install-nvidia-drivers.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -ex
+
+# Update nvidia drivers
+NVIDIA_DRIVERS_VERSION="375.51"
+
+wget "http://us.download.nvidia.com/XFree86/Linux-x86_64/${NVIDIA_DRIVERS_VERSION}/NVIDIA-Linux-x86_64-${NVIDIA_DRIVERS_VERSION}.run"
+sudo /bin/bash "./NVIDIA-Linux-x86_64-${NVIDIA_DRIVERS_VERSION}.run" -asq
+sudo reboot
```

Comment:
I don't know if it will mess up Packer, but the reboot seems to be required when installing the NVIDIA drivers. http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/accelerated-computing-instances.html#install-nvidia-driver